### PR TITLE
TKR: Map Plays and horn

### PR DESCRIPTION
**Pull request by AlexEKinney**
I've added map plays and the active horn

### CODE REVIEW

File: src/structures/MiniGames/TurboKartRacers.js [hunk 1]
```diff
@@ -41,6 +41,31 @@ class TurboKartRacers {
      * @type {number}
      */
     this.boxPickups = data.box_pickups || 0;
+    /**
+     * Horn 
+     * @type {string}
+     */
+    this.horn = data.horn || 0;
```
Comment (porterporter) [inline src/structures/MiniGames/TurboKartRacers.js]: horn as a string defaults to 0?

Comment (AlexEKinney) [reply src/structures/MiniGames/TurboKartRacers.js]: Yes but no
The username false results to 0 but everyone else is DEFAULT

Comment (AlexEKinney) [reply src/structures/MiniGames/TurboKartRacers.js]: I am good at pushing code. I like leaving my changes on my own device

**Comment by ChiefChippy2** [comment]
do you maybe have a list of possible horns so the typing can be more exact?

**Comment by AlexEKinney** [comment]
> do you maybe have a list of possible horns so the typing can be more exact?


DEFAULT
SHY
ALIEN
TAXI
KLAXON
TRICYCLE
ALARM
KLOON
TEDDY
TRUCK
JERRY

File: src/structures/MiniGames/TurboKartRacers.js [hunk 2]
```diff
@@ -41,6 +41,31 @@ class TurboKartRacers {
      * @type {number}
      */
     this.boxPickups = data.box_pickups || 0;
+    /**
+     * Horn 
+     * @type {string}
```
Comment (ChiefChippy2) [inline src/structures/MiniGames/TurboKartRacers.js]:
```suggestion
     * @type {'DEFAULT' | 'SHY' | 'ALIEN' | 'TAXI' | 'KLAXON' | 'TRICYCLE' | 'ALARM' | 'KLOON' | 'TEDDY' | 'TRUCK' | 'JERRY'}

```

File: typings/index.d.ts [hunk 3]
```diff
@@ -719,6 +719,11 @@ declare module 'hypixel-api-reborn' {
     silverTrophies: number;
     goldTrophies: number;
     boxPickups: number;
+    horn: string;
```
Comment (ChiefChippy2) [inline typings/index.d.ts]:
```suggestion
    horn: 'DEFAULT' | 'SHY' | 'ALIEN' | 'TAXI' | 'KLAXON' | 'TRICYCLE' | 'ALARM' | 'KLOON' | 'TEDDY' | 'TRUCK' | 'JERRY';
```

**Review by ChiefChippy2 — COMMENTED** [review]
Better typings...ig

**Pull request merged**